### DISCUSSION TITLE
JSON validation with Hibernate.

### DISCUSSION THREAD

**Discussion by chainhead** [quarkusio/quarkus#29305] [Q&A]
As a newbie to Quarkus, I am unable to follow what JSON document works successfully against the constraints defined in this [tutorial](https://quarkus.io/guides/validation). In the current form, unless the UI is launched and request traced in Developer Tools of a browser, I cannot decipher the JSON payload.

Can you please show me the JSON example?

[a comment by an automated account is not shown]

**Comment by gsmet** [comment]
I don't know what you're trying to do exactly. Hibernate Validator is a library to validate Java Beans and method calls. It doesn't support JSON validation at all (but can be used to validate the resulting bean if you deserialize your JSON content into a bean).

**Reply by chainhead** [reply]
That tutorial seems to consume a JSON payload at a REST end point. The tutorial doesn't describe how the JSON payload is expected to look like.

Only by looking at request in the Developer Tools in browser am I able to see the JSON payload.
